docs/contracts/functions.rst

```diff
@@ -102,10 +102,10 @@ two integers passed as function parameters, then you use something like:
         function arithmetic(uint _a, uint _b)
             public
             pure
-            returns (uint o_sum, uint o_product)
+            returns (uint oSum, uint oProduct)
```

Comment:
Same here. The `o_` is probably the convention used to indicate that it's an output parameter, i.e. used to return a value. Removing the underscore mangles it.